src/main/resources/templates/lib.ftl

```diff
@@ -442,7 +450,7 @@
 
 <#macro screenMessage value>
   <#if value?has_content>
-    <@variableLengthRepeatableField id="AF" value=value/>
+    <@variableLengthListField id="AF" value=value/>
```

Comment:
I'm not sure. There is no mention of what to do in this case. Since the field is repeatable, I suppose we could just split the message into multiple messages, like print line, just at a different limit, like the spec limit of 255 (assuming SCs adhere to this limit). That is what I had it doing prior to this PR, but the split was occurring at max print width, which is not correct according to the spec.

I guess an easy fix would be to pass the length to `variableLengthRepeatableField` and in the  `printLine` case, send max print width, and in `screenMessage` send 255. I think that is reasonable. I'll do that.